el-patch.el

```diff
@@ -640,12 +640,14 @@ is the Lisp form, read from the buffer at point."
                               ,@body))))
           (defun-buffer (car buffer-point))
           (defun-point (cdr buffer-point)))
-     (and defun-buffer
-          defun-point
-          (with-current-buffer defun-buffer
-            (save-excursion
-              (goto-char defun-point)
-              (read (current-buffer)))))))
+     (prog1 (and defun-buffer
+                 defun-point
+                 (with-current-buffer defun-buffer
+                   (save-excursion
+                     (goto-char defun-point)
+                     (read (current-buffer)))))
+       (when defun-buffer
+         (kill-buffer defun-buffer)))))
```

Comment:
I added this logic to kill the buffer of the source file. I think keeping the buffer open (for templates or patches) is unnecessary.

Reply:
Yeah, that seems good to me. Since we have the hack just above to ensure `el-patch` creates a new buffer rather than reusing an existing buffer, we know that this will not accidentally kill one of the user's currently open buffers, so it should be safe.